.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[env]
+#set stack size to 8MB
+RUST_MIN_STACK = "8388608"
```

Comment:
I am surprised that this helps. This should only affect child threads, while the main thread is never affected: [https://stackoverflow.com/a/29980945/3209904](https://stackoverflow.com/a/29980945/3209904)

Reply:
I see. The parser is running in a child thread. So it does help there, but not during type checking for example.

Reply:
Also fürs Typechecking hilfts aber, wie ich gerade noch mal ausprobiert habe, auch. Ohne das Setzen der Variable bekommt der `vector_add`-Test in `example_apps` einen Stackoverflow während des Typecheckings. Und wenn die Variable gesetzt ist, läuft der Test durch.

Reply:
Wobei jeder Test anscheinend in einem eigenen Nebenthread ausgeführt wird.